docs/reference/migration/migrate_7_0/restclient.asciidoc

```diff
@@ -20,4 +20,12 @@ e.g. `client.index(indexRequest, new Header("name" "value"))` becomes
 The Cluster Health API used to default to `shards` level to ease migration
 from transport client that doesn't support the `level` parameter and always
 returns information including indices and shards details. The level default
-value has been aligned with the Elasticsearch default level: `cluster`.
\ No newline at end of file
+value has been aligned with the Elasticsearch default level: `cluster`.
+
+=== Low-level REST client changes
+
+[float]
+==== Support for `maxRetryTimeout` removed from RestClient
+
+`RestClient` and `RestClientBuilder` no longer support the `maxRetryTimeout`
+setting.
```

Comment:
It'd be nice to explain why we removed it here.

Reply:
will add that